solc/CommandLineInterface.cpp

```diff
@@ -297,20 +274,19 @@ static bool needsHumanTargetedStdout(po::variables_map const& _args)
 		return true;
 	if (_args.count(g_argOutputDir))
 		return false;
-	for (string const& arg: {
-		g_argAbi,
-		g_argAsm,
-		g_argAsmJson,
-		g_argAstJson,
-		g_argBinary,
-		g_argBinaryRuntime,
-		g_argMetadata,
-		g_argNatspecUser,
-		g_argNatspecDev,
-		g_argOpcodes,
-		g_argSignatureHashes,
-		g_argStorageLayout
-	})
+	for (string const& arg:
+		 {g_argAbi,
+		  g_argAsm,
+		  g_argAsmJson,
+		  g_argAstJson,
+		  g_argBinary,
+		  g_argBinaryRuntime,
+		  g_argMetadata,
+		  g_argNatspecUser,
+		  g_argNatspecDev,
+		  g_argOpcodes,
+		  g_argSignatureHashes,
+		  g_argStorageLayout})
```

Comment:
I think that the `{` is supposed on the same line as `for` and `})` on a separate line.